src/cadenza.js

```diff
@@ -201,6 +209,7 @@ export class CadenzaClient {
    * @param {Geometry} [options.geometry] - A geometry to show on the map
    * @param {boolean} [options.hideMainHeaderAndFooter] - Whether to hide the main Cadenza header and footer
    * @param {boolean} [options.hideWorkbookToolBar] - Whether to hide the workbook toolbar
+   * @param {boolean} [options.expandNavigator] - Indicates if the navigator should be expanded.
    * @param {GlobalId} [options.highlightGlobalId] - The ID of an item to highlight / expand in the navigator
```

Comment:
Here exactly is what I thought: The highlightGlobalId is exactly expanding the navigator. So do we really need both parameters? And if yes, why? What's the example use case for it?

Reply:
no, highlightGlobalId does not expand the navigator, it just selects and expands the tree item.